modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -157,6 +157,7 @@ enum VideoCaptureProperties {
        CAP_PROP_FOCUS         =28,
        CAP_PROP_GUID          =29,
        CAP_PROP_ISO_SPEED     =30,
+       CAP_CROSSBAR_INPIN_TYPE      =31, //!<CrossBar input pin Setting
```

Comment:
Please add enum values to the end of list only.

Reply:
I have changed enum value to the end of list. thanks